### PR TITLE
February Beer and Tell 2016 and update template.

### PR DESCRIPTION
**Pull request by Osmose**
@stephaniehobson @willkg @craigcook @pmclanahan @peterbe @jpetto r?

This will be going up Wednesday morning so please leave your feedback by then. Thanks!

### CODE REVIEW

**Comment by stephaniehobson** [comment]
R+

**Comment by willkg** [comment]
Looks good to me.
r_plus

**Comment by pmclanahan** [comment]
R+ :cake:

**Comment by craigcook** [comment]
hulk-plus

File: beer_and_tell/template.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
+
 ## Name: Project
 Info about a project
 
 ---
 
-We did things!
-
-If you're interested in attending the next Beer and Tell, sign up for the
+aIf you're interested in attending the next Beer and Tell, sign up for the
```
Comment (peterbe) [inline beer_and_tell/template.md]: "aIf"

File: beer_and_tell/2016-02-February.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+application itself edits JSON files that are loaded by the web-based engine. The
+engine itself uses [pixi.js][] for rendering, and can be extended via plugins
+written in JavaScript.
+
+[Osmose]: https://mozillians.org/en-US/u/Osmose/
+[RPG Maker MV]: http://www.rpgmakerweb.com/
+[pixi.js]: http://www.pixijs.com/
+
+## Battleshits
+[peterbe][] stopped by to share [Battleshits][], a small web-based, fairly gross
```
Comment (peterbe) [inline beer_and_tell/2016-02-February.md]: Can you change "a small web-based" to "a mobile-friendly web app"?

File: beer_and_tell/2016-02-February.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+[Fanout]: https://fanout.io/
+[React]: https://facebook.github.io/react/
+
+## Chava
+Last up was [jpetto][] with a small personal project memorializing his local
+coffeeshop, [Chava][], which closed earlier this year. The page uses
+[Hammer.js][] for touch events and [LazyLoad][] to lazily load the images, but
+the lightbox implementation is custom-made from scratch. Neato!
+
+[jpetto]: https://github.com/jpetto
```
Comment (jpetto) [inline beer_and_tell/2016-02-February.md]: Just updated my mozillians account - [https://mozillians.org/en-US/u/jpetto/](https://mozillians.org/en-US/u/jpetto/). You know, for consistency.

Could mention [Gulp](http://gulpjs.com/) and the artisanal Python page generator? I just put the project up on [the GitHub](https://github.com/jpetto/chava), too.

**Pull request merged**